.github/workflows/galaxy.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish to Ansible Galaxy
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
If I'm not wrong, this is interesting specifically when pushing a new tag.

Reply:
Ok, I am not sure if the action takes into account the git releases, I can't find it in the docs, but I will add it, as it makes sense